.sonarcloud.properties

```diff
@@ -0,0 +1 @@
+
```

Comment:
Sonarqube files should not be included in the repository.

Reply:
removed the sonar files